include/robot_interfaces/n_joint_robot_types.hpp

```diff
@@ -48,6 +49,12 @@ struct NJointRobotTypes
         //! D-gain for position controller.  If NaN, default is used.
         Vector position_kd;
 
+        template <class Archive>
+        void serialize(Archive& archive)
+        {
+            archive(torque, position, position_kp, position_kd);
```

Comment:
similar question here: are we sure this does not allocate memory? @vincentberenz @MaximilienNaveau, i guess you guys had to deal with these kind of questions.

Reply:
@mwuethri Is it okay if we merge this for now?  In case there is really some memory allocation issue in the serialization, this would anyway be a more fundamental issue that would not be fixed in this PR.

Reply:
My understanding of cereal : it is highly templated exactly to avoid runtime memory allocation